airgun/entities/contentview.py

```diff
@@ -4,43 +4,68 @@
 from airgun.navigation import NavigateStep, navigator
 from airgun.views.contentview import (
     AddNewPuppetModuleView,
+    ContentViewCopyView,
     ContentViewCreateView,
     ContentViewEditView,
+    ContentViewRemoveView,
     ContentViewTableView,
+    ContentViewVersionDetailsView,
     ContentViewVersionPromoteView,
     ContentViewVersionPublishView,
+    ContentViewVersionRemoveConfirmationView,
+    ContentViewVersionRemoveView,
     SelectPuppetModuleVersionView,
 )
 
 
 class ContentViewEntity(BaseEntity):
 
     def create(self, values):
+        """Create a new content view"""
         view = self.navigate_to(self, 'New')
         view.fill(values)
         view.submit.click()
 
     def delete(self, entity_name):
-        view = self.navigate_to(self, 'Edit', entity_name=entity_name)
-        view.actions.fill('Remove Content View')
-        view.dialog.confirm()
+        """Delete existing content view"""
+        view = self.navigate_to(self, 'Delete', entity_name=entity_name)
+        if view.conflicts_present:
+            raise AssertionError(
+                'Unable to delete content view. Following conflicts are'
+                'present: {}'.format(view.table.read())
+            )
```

Comment:
is there a specific reason to raise explicitly assert error,
assert key word is not only for test frameworks 
```python
assert not view.conflicts_present, 'Unable to delete content view. Following conflicts are'
                'present: {}'.format(view.table.read())
```
what do you think?